WooCommerce/Classes/ViewRelated/Blaze/ConfirmPayment/BlazeConfirmPaymentViewModel.swift

```diff
@@ -232,6 +245,29 @@ private extension BlazeConfirmPaymentViewModel {
     }
 }
 
+private extension BlazeConfirmPaymentViewModel {
+    enum Localization {
+        static let totalWeeklyAmount = NSLocalizedString(
+            "blazeConfirmPaymentViewModel.totalWeeklyAmount",
+            value: "%1$@ weekly",
+            comment: "Total weekly amount of an unlimited Blaze campaign. " +
```

Comment:
Suggestion: I think we can elaborate on the "unlimited" wording. 

```suggestion
            comment: "Total weekly amount of a Blaze campaign configured to run forever with no end date. " +
```

Applies to `totalWeeklyAmountWithCurrency` string as well.